### PR TITLE
Adds support for variable path on twig tag inlinecss

### PR DESCRIPTION
**Pull request by cordoval**
Addresses https://github.com/gushphp/ToInlineStyleEmailBundle/issues/26

### CODE REVIEW

**Comment by cordoval** [comment]
@kalypso63 can you please send me your fix for the coding style as a PR to this PR? thanks 👍

**Comment by kalypso63** [comment]
@cordoval the fix for coding style for README.md ? I m not available during 1 week. Can you do it without me ?

**Comment by cordoval** [comment]
@kalypso63 i will do it for you 👍

**Comment by kalypso63** [comment]
Hi @cordoval. Could you advance on this point ?

**Comment by cordoval** [comment]
Yes i fell sick but i will come back to this

**Comment by cordoval** [comment]
@kalypso63 retaking this now

File: Twig/InlineCssNode.php [hunk 1]
```diff
@@ -25,16 +25,29 @@ public function __construct(\Twig_Node $body, $css, $lineno = 0, $debug, $tag =
...
+                $css = '"' . addslashes(file_get_contents($this->getAttribute('css'))) . '"';
+            }
+            $compiler->addDebugInfo($this)
+                ->write("ob_start();\n")
+                ->subcompile($this->getNode('body'))
+                ->write(sprintf('echo $context["inlinecss"]->inlineCSS(ob_get_clean(), %s);' . "\n", $css))
+            ;
         } else {
-            $css = '"' . addslashes(file_get_contents($this->getAttribute('css'))) . '"';
+            //get path of css
```
Comment (cordoval) [inline Twig/InlineCssNode.php]: @kalypso63 what happens with the debug here? or we are not missing things?

Comment (kalypso63) [inline Twig/InlineCssNode.php]: Actually nothing happens with debug and dynamic variable at this part of code

**Comment by cordoval** [comment]
going with my gut feeling 👍

**Pull request merged**